core/include/mm/core_mmu.h

```diff
@@ -567,26 +567,41 @@ struct tee_mmap_region *
 core_mmu_find_mapping_exclusive(enum teecore_memtypes type, size_t len);
 
 /*
- * tlbi_mva_range() - Invalidate TLB for virtual address range
+ * tlbi_va_range() - Invalidate TLB for virtual address range
  * @va:		start virtual address, must be a multiple of @granule
  * @len:	length in bytes of range, must be a multiple of @granule
  * @granule:	granularity of mapping, supported values are
  *		CORE_MMU_PGDIR_SIZE or SMALL_PAGE_SIZE. This value must
  *		match the actual mappings.
  */
-void tlbi_mva_range(vaddr_t va, size_t len, size_t granule);
+#if defined(ARM32) || defined(ARM64)
+static inline void tlbi_va_range(vaddr_t va, size_t len, size_t granule)
+{
+	tlbi_mva_range(va, len, granule);
```

Comment:
I meant: why not get rid of `tlbi_mva_range()`?

Even so, this PR leaves `_mva` in many places (including in `core/arch/riscv/mm/tlb_helpers_rv.S`):
```
$ git grep _mva | wc -l
38
```
If `va` is not a good term to begin with, even on modern Arm, I think that *all* occurrences should be replaced by `va`.

Reply:
OK, I can submit more commits to replace `mva` with `va`.

Reply:
Hi @jforissier 

I made a commit for this, please check it.
Note that some "mva" are still reserved for ARM32.